website/tests/pages/submission/template.spec.ts

```diff
@@ -37,4 +46,12 @@ test.describe('The submit page', () => {
             readable.on('end', () => resolve(data));
         });
     }
+
+    function skipDownloadTestInWebkit(browserName: 'chromium' | 'firefox' | 'webkit') {
+        test.skip(
+            browserName === 'webkit',
+            'webkit seems to ignore the content disposition header.\n' +
```

Comment:
```suggestion
            'webkit seems to ignore the content disposition header.\n' +
```
I'm not 100% sure but I think from last time it might be just playwright-webkit rather than Safari.